### PR TITLE
Toys expansion

### PR DESCRIPTION
**Pull request by dwinters99**
I was asked to make a clown overhaul, and I got carried away making toys so now this is a toy overhaul. _Clown overhaul coming soon._ 

I still have a few things to do but I either don't know how to do them or don't want to. This includes making the gnome announce when it's brought to centcomm (little rocket man) and the cat toys making cat subtypes follow a mob that holds them. 

These are all obtainable either in loadout, the arcade machines, or both.

At some point in the future I'll be porting the ticket redemption machine from Paradise which allows you to convert arcade machines to award tickets that can be redeemed for specific items.

**EDIT:** If you have anything you want me to add, PLEASE tell me. Making this was some of the most fun I've had in a long time on account of the challenges required to make these things work in the specific way they needed to work.
### Plushies:
---
- IPC plushie (paradise port)
_Use any subtype of food/snacks/slice/bread or bread tubes to make toast. Purely visual._
image
- Toaster plushie
 _Subtype of IPC plushie. Includes the toaster functionality._
image

- Snake plushie (paradise port)
image

- Marketable CRO plushie
_Swipe your ID and listen to it talk._
image

- Perfectly Generic Plushie
image

- Moth plushie
_Squeeze it to hear authentic mothman horror._
image

- Crab plushie
image

- Possum plushie
image

- Goose plushie
image

- White mouse plushie
image

- Pet rock plushie
_Get a pen and draw a face on it. You can even put a bow on it._
image

### Animal toys
---
- Chewtoys
_Nice and squeaky. Also includes colorable versions._
image

- Cat toys
image

### Novelties
---
- Toy flash
_The bulb is too weak to blind anyone, but it still makes a satisfying noise._
image

- Big red button (paradise port)
_DO NOT PRESS. Don't press it. Don't do it. DO NOT._
image

- Garden gnome
image

- Toy AI
_Ask it to state laws and shake it for some computer humor._
image

- Buzzer ring
_Remember to take it off before shaking the captain's hand._
image

- Fake handcuffs
_For playing cops and robbers. Comes with a safety latch to quick-release them._
image

- Nuclear Fission Explosive (paradise port)
_An essential toy for any young aspiring Heavy Asset Protection Commando._
image

- Minigibber (paradise port)
_Stick a certified Space Life™ figure in and press the button. Note: Refuses organic compounds._
image

- Toy Alien (paradise port)
_Pull the string for a genuine hiss, straight from V4._
image

- Russian Revolver
_Spin the cylinder and pull the trigger. You have a 5/6 chance of winning. Note: Does not contain real bullets._
image

- Trick revolver
_A party popper shaped like a gun. Made of metal and wood, it's hard to tell it from the real thing._
image

- Toy chainsaw
_Made of foam. Pull the string and put fear into the hearts of diona everywhere._
image

- Snake popper
_Make sure to check your bread. Opening this bread tube will pop out a snake plushie. Almost always._
image
### Misc
- Added a new quack for rubber ducks
- Fixes a typo in plush box description
- Added confetti based on sparks

### CODE REVIEW

File: code/game/objects/effects/confetti.dm [hunk 1]
```diff
@@ -0,0 +1,42 @@
+/obj/effect/effect/sparks/confetti
```
Comment (Novacat) [inline code/game/objects/effects/confetti.dm:1]: This should probably be marked with a _vr, even if it is not an override it is useful to know what files are vorestation-specific and which are not.

**Comment by TheFurryFeline** [comment]
Nice with the PGP, ref of Starbound? :P

**Comment by dwinters99** [comment]
> Nice with the PGP, ref of Starbound? :P

Actually, a reference to what Starbound was referencing. You know, the webcomic that everyone liked when they were 14 and that I refuse to believe anyone over the age of 20 still likes. H*mestuck.

**Comment by TheFurryFeline** [comment]
> > Nice with the PGP, ref of Starbound? :P
> 
> Actually, a reference to what Starbound was referencing. You know, the webcomic that everyone liked when they were 14 and that I refuse to believe anyone over the age of 20 still likes. H*mestuck.

Oh, so it was a ref already in Starbound, huh. Neato.

**Comment by Iroquois-Pliskin** [comment]
Only issues I have are with the toaster plushie and CRO plushie. Character/player references are kinda meh. The rest seem neat.

**Review by Novacat — APPROVED** [review]
Code-wise this is fine, while I am not particularly fussed about the references I will leave this up till the next patch to gather more feedback.

**Comment by Novacat** [comment]
Thinking further on this, we already have a Resh plushie so... this should not be a problem.

**Pull request merged**